README.md

```diff
@@ -1,30 +1,34 @@
-# Pendulum-Ink-Wrapper
+# Pendulum-Solidity Wrappers
 
-A collection of ink! contracts that interact with Pendulum's chain extensions.
+A collection of contracts written in Solidity and compiled to WASM using Solang compiler for deployment on the
+Contracts pallet.
 
-## Setup
+These contracts implement well know interfaces (`IERC20`, `IPriceOracleGetter`), but the novel feature
+is the ability to call Pendulum's chain extensions which allows them to interact with 
+the logic of the pallets. Notably, fetching latest price of assets from the chain and interactions with the tokens pallet.
 
-cargo install cargo-contract --version 2.1.0
+Please see 'docs' for a more information.
 
-## Build
+# Compiling the solidity contract
 
-`cargo contract build`
+### Install solang
 
-## Deploy
+```bash
+brew install hyperledger/hyperledger/solang
+```
 
-UI to deploy and use on Foucoco testnet: https://contracts-ui.substrate.io/?rpc=wss://rpc-foucoco.pendulumchain.tech
+### Compile the contract
 
-Contract specific instructions to deploy and use:
+```bash
+solang compile --target polkadot {path/to/contract}.sol
+```
 
-[price_feed](/price_feed/README.md)
+### Deploying the contract
 
-[psp_pendulum](/price_feed/README.md)
+As a result you will get a file called `{contract}.contract`.
+You can now deploy this contract to a parachain using Contracts UI.
```

Comment:
I think we can remove this section from this README as it's already covered in the respective READMEs in the sub directories.